### ISSUE TITLE
Feature Request: alicloud_image - add the platform "Windows Server 20xx" support

### ISSUE THREAD

**Issue by tongyond** [aliyun/terraform-provider-alicloud#5795]
Hi Support,

### Terraform Version
```hcl
Terraform v1.2.9
on linux_amd64
+ provider registry.terraform.io/aliyun/alicloud v1.193.0
+ provider registry.terraform.io/hashicorp/alicloud v1.193.0
+ provider registry.terraform.io/hashicorp/local v2.2.3
+ provider registry.terraform.io/hashicorp/random v3.4.3
```

### Affected Resource(s)
- alicloud_image

### Terraform Configuration Files
```hcl
resource "alicloud_image" "default" {
  ...
  architecture = "x86_64"
  platform     = "Windows Server 2022"
  ...
}
```

### Debug Output

```hcl
│ Error: expected platform to be one of [CentOS Ubuntu SUSE OpenSUSE RedHat Debian CoreOS Aliyun Linux Windows Server 2003 Windows Server 2008 Windows Server 2012 Windows 7 Customized Linux Others Linux], got Windows Server 2022
```

### Expected Behavior
Please help to add "Windows Server 2016" "Windows Server 2019" "Windows Server 2022" as a valid value of platform of alicloud_image module

### Actual Behavior
"Windows Server 2016" "Windows Server 2019" "Windows Server 2022" is not list in the valid value of platform of alicloud_image module

### Steps to Reproduce
1. `terraform plan` or `terraform apply`

### References
I have also opened a issues on https://github.com/terraform-alicloud-modules/terraform-alicloud-ecs-image/issues/6 , but no respond, so i have to open a issue here again.
https://registry.terraform.io/providers/aliyun/alicloud/latest/docs/resources/image

**Comment by xiaozhu36** [comment]
HI @tongyond This issue has been fixed by version 1.197.0. Please upgrade it and have a check.

**Comment by xiaozhu36** [comment]
Closed for no reply. Please reopen it if there are more questions.
